Description: Changes to make php use versioned paths and other minor
 cleanup changes.
Origin: vendor
Forwarded: not-needed
Last-Update: 2018-11-17

---
 configure.in          |    3 ++-
 scripts/php-config.in |    4 ++--
 2 files changed, 4 insertions(+), 3 deletions(-)

--- a/configure.in
+++ b/configure.in
@@ -1013,7 +1013,7 @@ if test "$PHP_CLI" = "no"; then
 fi
 
 PHP_ARG_WITH(pear, [whether to install PEAR],
-[  --with-pear=DIR         Install PEAR in DIR [PREFIX/lib/php]
+[  --with-pear=DIR         Install PEAR in DIR [PREFIX/share/php]
   --without-pear          Do not install PEAR], DEFAULT, yes)
 
 if test "$PHP_PEAR" != "no"; then
@@ -1471,6 +1471,7 @@ $php_shtool mkdir -p scripts/man1
 ALL_OUTPUT_FILES="php5.spec main/build-defs.h \
 scripts/phpize scripts/man1/phpize.1 \
 scripts/php-config scripts/man1/php-config.1 \
+ext/ext_skel
 $PHP_OUTPUT_FILES"
 
 dnl
--- a/scripts/php-config.in
+++ b/scripts/php-config.in
@@ -6,10 +6,10 @@ exec_prefix="@exec_prefix@"
 version="@PHP_VERSION@"
 vernum="@PHP_VERSION_ID@"
 include_dir="@includedir@/php"
-includes="-I$include_dir -I$include_dir/main -I$include_dir/TSRM -I$include_dir/Zend -I$include_dir/ext -I$include_dir/ext/date/lib"
+includes="-I$include_dir -I$include_dir/main -I$include_dir/TSRM -I$include_dir/Zend -I$include_dir/ext -I$include_dir/ext/date/lib $(getconf LFS_CFLAGS)"
 ldflags="@PHP_LDFLAGS@"
 libs="@EXTRA_LIBS@"
-extension_dir='@EXTENSION_DIR@'
+extension_dir="@EXTENSION_DIR@"
 man_dir=`eval echo @mandir@`
 program_prefix="@program_prefix@"
 program_suffix="@program_suffix@"
